### DISCUSSION TITLE
Strategies for shipping/publishing chakra

### DISCUSSION THREAD

**Discussion by kamleshchandnani** [chakra-ui/chakra-ui#4232] [Q&A]
I wanted to ask, if the strategy of shipping chakra is documented anywhere? Like I would like to understand what was the reason of just transpiling chakra with babel and publishing instead of bundling it using rollup or something else.

For more context you can read the discussions in [this tweet](https://twitter.com/_kamlesh_/status/1404701659180134400)

**Comment by kamleshchandnani** [comment]
@segunadebayo Maybe you're the best person to share detailed insights on this.

**Comment by Brennvo** [comment]
Also very curious about this, too.
